easybuild/easyconfigs/g/GCCcore/GCCcore-6.3.0-fix-ubsan.patch

```diff
@@ -0,0 +1,12 @@
+diff -ru gcc-6.3.0-orig/gcc/ubsan.c gcc-6.3.0/gcc/ubsan.c
```

Comment:
@zao Please mention source/author for this patch above this line, and maybe also a short description of the problem it fixes (& how)